pallets/parachain-staking/src/lib.rs

```diff
@@ -252,7 +255,7 @@ pub mod pallet {
 	pub enum Event<T: Config> {
 		/// Started new round.
 		NewRound {
-			starting_block: BlockNumberFor<T>,
+			starting_block: u64,
```

Comment:
Yes I agree, but as we are not implementing the feature in Moonbeam and Moonriver yet, it would be confusing if we rename the field to "slot" now, as in those chains it will continue to be the block number. Once it is enabled on all chains, we can rename the field.